src/actions/Entities/VaccinePrescriptionActions.js

```diff
@@ -94,6 +95,30 @@ const confirm = () => (dispatch, getState) => {
     });
     prescription.finalise(UIDatabase);
   });
+};
+
+const createRefusalNameNote = name => {
+  const [patientEvent] = UIDatabase.objects('PatientEvent').filtered('code == "RV"');
+  const id = generateUUID();
+  const _data = JSON.stringify({ refused: true, date: new Date() });
```

Comment:
Just didn't know what to put in the `data` 🤣 
and since the `entryDate` isn't showing in desktop thought I'd pop it in. Just leave the `_data` empty?

Reply:
Hmm..... yeah it's weird having always some data element with just "true" also, huh.

Entry date bug is just this line I think https://github.com/openmsupply/mobile/blob/2b960be6a12e77aa9032793e3add239ea35cf450/src/sync/outgoingSyncUtils.js#L414

so I guess an empty _data is all good...?!